### DISCUSSION TITLE
creating .env file from node ace command

### DISCUSSION THREAD

**Discussion by ahmed2104** [adonisjs/core#1332] [Help] [answered]
how to create .env file from node ace command after push source code to github and clone it on my pc

**Comment by thetutlage** [comment, marked as answer]
Why need an ace command for this? Simply say `touch .env`, or better `cp .env.example .env`

**Reply by ahmed2104** [reply]
my .env.example is empty :D

**Reply by thetutlage** [reply]
Then how come a command can fill values for you? You have to self insert the values in your .env file. We do not know your database password or current environment

**Reply by ahmed2104** [reply]
I just mean the default setup
